src/memory_model/src/guest_memory.rs

```diff
@@ -468,6 +435,14 @@ impl Bytes<GuestAddress> for GuestMemory {
                 .map_err(|e| Error::MemoryAccess(addr, e))
         })
     }
+
+    fn read_obj<T: ByteValued>(&self, addr: GuestAddress) -> std::result::Result<T, Self::E> {
```

Comment:
Same here for documentation and example comments

Reply:
Done.